tests/run_test.go

```diff
@@ -21,7 +21,7 @@ func runOneToOneTest(tb testing.TB, options *serverOptions, tests func(*testConn
 func runOneToOneTestWithAuth(tb testing.TB, options *serverOptions, tests func(*testConnection, *testSession)) {
 	runOneToOneTest(tb, options, func(c *testConnection, s *testSession) {
 		withTag(func(tag string) {
-			c.Cf("%v login %v %v", tag, options.defaultUsername(), options.defaultUserPassword()).OK(tag)
+			c.Cf("%v login %v %s", tag, options.defaultUsername(), string(options.defaultUserPassword())).OK(tag)
```

Comment:
my point was, if you use `%s`, you don't need to convert bytes to string before printing :p

```suggestion
			c.Cf("%v login %v %s", tag, options.defaultUsername(), options.defaultUserPassword()).OK(tag)
```

Reply:
:facepalm: